include/core.hpp

```diff
@@ -33,7 +33,9 @@ namespace color_coded
 
     inline std::string temp_dir()
     {
-      static auto const dir(fs::temp_directory_path() / "color_coded/");
+      static auto const dir(fs::temp_directory_path() /
+                            (std::string("color_coded-") +
+                             std::string(getenv("USER"))));
```

Comment:
There's one big issue with this, since `getenv` can return `nullptr` and constructing a `std::string` from `nullptr` is undefined behavior. I think we should pull out the `getenv` into a separate `static auto const` and then use a ternary for the concatenation.